Source/Client/Sound/NAudioPlaybackEngine.cs

```diff
@@ -33,13 +33,16 @@ public ISampleProvider ConvertToRightChannelCount(ISampleProvider input)
             input = resampler.ToSampleProvider();
         }
 
-        if (input.WaveFormat.Channels == _mixer.WaveFormat.Channels)
+        if (input.WaveFormat.Channels == 2)
```

Comment:
`PanningSampleProvider` accepts mono input, so we need to return early

Reply:
And currently, stereo inputs are only music tracks?

Reply:
Now wait, did you mean `!= 2`? Otherwise, I don't understand this.

Reply:
Nope, if we have stereo sound (currently those are only music tracks), we cannot pipe it to the panning sample provider, because it throws on non-mono input. So we simply return the track if it is stereo sound.

For mono sounds, we pipe them to panning sample provider, which underneath converts them to stereo with possibility to control pan

Reply:
Ah, now I understand.